mat/eigen.go

```diff
@@ -104,6 +104,15 @@ func (m *Dense) EigenvectorsSym(e *EigenSym) {
 	m.Copy(e.vectors)
 }
 
+type EigenFact int
```

Comment:
I think the type name is not accurate, I would reflect in it that it concerns the eigenvectors. Perhaps `EigenVect` or similar?